src/values/ObjectValue.js

```diff
@@ -55,7 +55,7 @@ import * as t from "babel-types";
 import invariant from "../invariant.js";
 
 export default class ObjectValue extends ConcreteValue {
-  constructor(realm: Realm, proto?: ObjectValue | NullValue, intrinsicName?: string) {
+  constructor(realm: Realm, proto?: ObjectValue | NullValue, intrinsicName?: string, refuseSerialization?: true) {
```

Comment:
Why should the default value to be "true"? By default all object/properties should refuse serialization?

Reply:
I think that means if the parameter is not passed, the default value is `undefined`. However, if an argument value is passed, it must be `true`. Maybe a bit surprising, but correct.

Reply:
Hmm, really confusing code style in my opinion. I would suggest change to:
```js
refuseSerialization = false: boolean
```
which is much more readable.

Reply:
This is supposed to mean that the value is either true or undefined because it doesn't make sense to specify it as false when it defaults to false, I guess that's not clear so I can change this to boolean.